clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -762,6 +775,12 @@ protected void handleRoleCommand() {
     }
   }
 
+  private String getOneRole(String[] roles, String command) {
+    Preconditions.checkArgument(
+        roles.length == 1, command + " requires only one role, but multiple are currently passed.");
```

Comment:
I think what Justin suggested was that to check if this actually happens.
If the `--role` option is supposed to be specified only once, would it be possible for a users to specify that option for multiple times?
If the answer is no, we don't need this check.
It the answer is yes, we have a serious problem regarding command line option checking. I mean, we have too many cases to check.

Reply:
@tengqm `gcli role details -m demo_metalake --role roleA roleB`, The check is meant to prevent such example. IHMO, We should keep this check.

Reply:
@tengqm `gcli role details -m demo_metalake --role roleA roleB`, The check is meant to prevent such example. IHMO, We should keep this check.

Reply:
Got it. But this is weird.
The command line will automagically interpret everything that appear after `--role` as the argument value for `--role`?
I was thinking that if you want to specify multiple values for a single command, the values must be a comma-separated string like this, `--role roleA,roleB`.
Maybe I'm wrong.

Reply:
By default, the CLI uses space as a demiliter, or you can specify multiple options via `--option opt1 --option opt2 --option opt3`. However, that is only for options that can accept multiple values.

Reply:
My question about this line of code is whether it is user-friendly. i.e. what error is displayed and in what form to the user if this precondition fails?